lib/sda/tree_test.go

```diff
@@ -472,6 +472,46 @@ func TestEntityList_Publics(t *testing.T) {
 // - public keys
 // - corner-case: accessing parent/children with multiple instances of the same peer
 // in the graph
+//BenchmarkTreeMarshal will be the benchmark for the conversion between TreeMarshall and Tree
+//It takes 37461141 ns before optimization on Macbook Pro
```

Comment:
Now that we don't do the optimisation, we might get rid of that line.

We make comments starting with a space, and the above comments (line 472-474) can be deleted (don't know where they come from).